mne_bids/tests/test_write.py

```diff
@@ -56,16 +57,29 @@
 # using the stable bids-validator and make a direct call of bids-validator
 # also: for windows, shell = True is needed to call npm, bids-validator etc.
 # see: https://stackoverflow.com/q/28891053/5201771
-shell = False
-bids_validator_exe = ['bids-validator']
-if platform.system() == 'Windows':
-    shell = True
-    exe = os.getenv('VALIDATOR_EXECUTABLE', 'n/a')
-    if 'VALIDATOR_EXECUTABLE' != 'n/a':
-        bids_validator_exe = ['node', exe]
+@pytest.fixture(scope="session")
+def _bids_validate():
+    shell = False
+    config_path = _TempDir()
+    config_fname = op.join(config_path, 'validator_config.json')
+    with open(config_fname, 'w') as f:
+        json.dump(dict(error=["NIFTI_UNIT"]), f)
```

Comment:
why do we want to ignore this error by default? Ignoring errors is a bad idea, because MNE-BIDS will start producing invalid datasets

Reply:
we are not ignoring this error. In fact, we are raising the status of something that was a warning to be an error. Not sure if we want all warnings to be errors, but at least for some, we do

Reply:
ah I see, +1